ui/src/views/authentication/unauthenticated/login/index.js

```diff
@@ -56,6 +69,7 @@ class Login extends Component {
 									onChange={this.handleInput}
 									required
 								/>
+								<div className="error-text">{this.state.error}</div>
```

Comment:
the material ui library that i've used supports error text on the TextField. [https://material-ui-next.com/demos/text-fields/](https://material-ui-next.com/demos/text-fields/) so it will look consistent with the rest of the app.

Reply:
Sure man, so you want the actual text fields to have the error? That doesn't really work for the invalid username/password fields as the error is about both fields.

Reply:
I did think about that, I think I expect both fields to be in error state saying the email and password don't match or invalid credentials which would mean it stays looking very material designy. But I can see where you're coming from the only thing is it won't be very consistent without a fair bit of styling

Reply:
Just pushed a change - both will show the right error class, removed the custom one. But I've also kept the message down the bottom to show it's an error with both fields.